demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -140,45 +147,60 @@
     ProductMilestone demoProductMilestone1;
 
     User demoUser;
+
     User pncAdminUser;
 
     public void verifyData() {
         // Check number of entities in DB
         Preconditions.checkState(projectRepository.count() > 0, "Expecting number of Projects > 0");
         Preconditions.checkState(productRepository.count() > 0, "Expecting number of Products > 0");
-        Preconditions.checkState(buildConfigurationRepository.count() > 0, "Expecting number of BuildConfigurations > 0");
-        Preconditions.checkState(productVersionRepository.count() > 0, "Expecting number of ProductVersions > 0");
-        Preconditions.checkState(buildConfigurationSetRepository.count() > 0, "Expecting number of BuildRepositorySets > 0");
-        Preconditions.checkState(artifactRepository.count() > 0, "Expecting number of Artifacts > 0");
+        Preconditions.checkState(buildConfigurationRepository.count() > 0,
```

Comment:
Just a note, I see you didn't write this:
It's always good to print the value that didn't pass the assertion